Python_hw1/hw1.py

```diff
@@ -0,0 +1,20 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = str(input("Введите трехзначное число: "))
+if len(str(a)) < 3 or len(str(a)) > 3:
+  print("Число не трехзначное! Введите число, состоящее из трёх цифр: ")
+  a = str(input())
+if len(str(a)) == 3: 
+  a = int(a)
+  b = 0
+  b = a % 10
+  a = a // 10
+  c = a % 10
+  a = a // 10
+  d = a % 10
```

Comment:
Нужно значительно проще.
```suggestion
  c = a // 10 % 10
  d = a // 100
```